setup.py

```diff
@@ -15,7 +15,7 @@
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'cycler',
                       'numpy', 'matplotlib', 'super_state_machine',
-                      'historydict'],
+                      'historydict', 'event_model'],
```

Comment:
I am with Dan, if this were wrong I would expect the install to fail, but if this is right, there is evil magic / heuristics going on